src/compas/robots/model/robot.py

```diff
@@ -290,7 +290,7 @@ def load_geometry(self, *resource_loaders, **kwargs):
         force = kwargs.get('force', False)
 
         loaders = list(resource_loaders)
-        loaders.insert(0, DefaultMeshLoader())
+        loaders.append(DefaultMeshLoader())
```

Comment:
I don't think this is correct, the `DefaultMeshLoader` should be inserted at the first position so that it's really the default one, otherwise other loaders will take precedence. Is there a reason why this should change?

Reply:
My understanding is as follows: 
If the user specifies `resource_loaders` (I presume that by default, no resource loader is passed in), why wouldn't the user-defined `resource_loaders` would be processed in first?

Reply:
not sure if this exactly applies to the implementation here, but generally speaking things should be processed in the opposite order they inherit. so default first, user-specified later. that way defaults are present and can be overwritten by more specific implementations...

Reply:
Fair enough. I will undo this part!